pkgs/tools/filesystems/bcachefs-tools/default.nix

```diff
@@ -72,6 +73,11 @@ stdenv.mkDerivation (finalAttrs: {
   # FIXME: Try enabling this once the default linux kernel is at least 6.7
   doCheck = false; # needs bcachefs module loaded on builder
 
+  patches = [ (fetchpatch {
```

Comment:
Please add a comment what the patch does and at which upstream release it will likley be resolved

Reply:
Also link the upstream issue.

Reply:
@onny I can't speak on behalf of upstream; but I'd presume this will be resolved with the next point release?